scripts/makecrates.py

```diff
@@ -220,6 +222,8 @@ def main(devices_path, yes, families):
         yamlfile = os.path.basename(path)
         family = re.match(r'stm32[a-z]+[0-9]', yamlfile)[0]
         device = os.path.splitext(yamlfile)[0].lower()
+        if family == 'stm32wle5':
```

Comment:
```suggestion
        if family.startswith('stm32wl'):
```

This should be more future-proof.